src/v2/guide/components-props.md

```diff
@@ -330,12 +330,14 @@ Vue.component('base-input', {
 })
 ```
 
+<p class="tip">Note that `inheritAttrs: false` option does **not** affect `style` and `class` bindings.</p>
+
 This pattern allows you to use base components more like raw HTML elements, without having to care about which element is actually at its root:
 
 ```html
 <base-input
-  v-model="username"
-  class="username-input"
-  placeholder="Enter your username"
+  v-model="email"
```

Comment:
Why this change?

Reply:
Original example had two non-inherited attributes, so I'd want to keep this count. I think that adding a custom `type` attribute made sense in this specific example (and added small cognitive load).

I can partially revert it, removing problematic `class` and keeping only `placeholder` attribute.

Reply:
I'm replacing the example above, too, since it's plainly wrong (neither `class` nor `style` keys ever appear in `$attrs` binding).